main.cpp

```diff
@@ -570,6 +823,25 @@ Filter Synthesiser::applyFilterToSound(double cutoff)
     return cutoffValue;
 }
 
+void Synthesiser::chordTypeSelector(std::string mode)
+{
+    std::map<std::string, std::vector<int> > chordType
+            {
+                    {"major", {0, 4, 7}},
+                    {"minor", {0, 3, 7}},
+                    {"diminished", {0, 3, 6}},
+                    {"augmented", {0, 4, 8}}
+            };
```

Comment:
Fix the indentation
- [ ] revise

Reply:
You did not fix the indentation
Fix the indentation
Stop relying on the auto-formatter in your IDE.
- [ ] revise